packages/flutter/test/cupertino/route_test.dart

```diff
@@ -1624,17 +1594,17 @@ class MockNavigatorObserver extends NavigatorObserver {
   final List<NavigatorInvocation> invocations = <NavigatorInvocation>[];
 
   @override
-  void didStartUserGesture(Route<Object> route, Route<Object> previousRoute) {
+  void didStartUserGesture(Route<dynamic> route, Route<dynamic>? previousRoute) {
```

Comment:
here and below: use `Object?` instead of `dynamic` if possible.

Reply:
The analyzer dings me for it. It only passes if I match the signature of the super class.

Reply:
Ahhh ok. We may want to separately consider having that be `Object?` instead of `dynamic`, since dynamic introduces terror into the type analysis.

Reply:
(Again, I'm not clear on whether there's some good reason to not do that based on other code though)